tests/final_chain_test.cpp

```diff
@@ -277,6 +278,36 @@ TEST_F(FinalChainTest, nonce_skipping) {
   advance({std::make_shared<Transaction>(5, 13, 0, TRX_GAS, dev::bytes(), key.secret(), key.address())});
 }
 
+TEST_F(FinalChainTest, initial_validators) {
+  const dev::KeyPair key = dev::KeyPair::create();
+  const std::vector<dev::KeyPair> validator_keys = {dev::KeyPair::create(), dev::KeyPair::create(),
+                                                    dev::KeyPair::create()};
+  cfg.state.genesis_balances = {};
+  cfg.state.dpos.emplace();
+  cfg.state.dpos->eligibility_balance_threshold = 100000;
+  cfg.state.dpos->vote_eligibility_balance_step = 10000;
+  cfg.state.dpos->maximum_stake = 10000000;
+  cfg.state.dpos->minimum_deposit = 1000;
+  cfg.state.dpos->eligibility_balance_threshold = 1000;
```

Comment:
lets put everywhere real values as we had problems with dpos tests when they passed just because the data we set there were nonsense...
Every value here should be multiplied by 10^18